idea-plugin/src/main/java/com/palantir/javaformat/intellij/PalantirJavaFormatFormattingService.java

```diff
@@ -61,7 +61,7 @@ protected FormattingTask createFormattingTask(@NotNull AsyncFormattingRequest re
 
     @Override
     public @NotNull Set<Feature> getFeatures() {
-        return Set.of(Feature.FORMAT_FRAGMENTS, Feature.OPTIMIZE_IMPORTS);
+        return Set.of(Feature.FORMAT_FRAGMENTS);
```

Comment:
Should we return an empty set (or not override if it's set up like that) `getImportOptimizers`?